purchase_all_shipments/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Purchase All Shipments
+======================
+
+With the core "purchase" module, in a purchase order a button "In Shipments"
+lets the user see the picking that was generated by the order itself.
+
+With this module, that button is replaced by an "All Shipments" button that
+shows the original picking, and all others that are grouped with it. This
+should include pickings associated to the procurements that generated the
+purchase, and also pickings that have been chained with push rules.
+
+This is consistent with the "sale" module, where from the sale order the user
+can access the generated delivery and all chained ones.
+
+The implementation uses the procurement group of the moves in the generated
+picking. The procurement group is always present in purchases, also when there
+is no procurement.
+
+Configuration
+=============
```

Comment:
Remove Configuration section if this one is empty. 

Not mandatory but you could use oca-gen-addon-readme to generate the new readme OCA format, this would clean up this file for you. review https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst